pkg/term/term.go

```diff
@@ -25,6 +26,20 @@ type Winsize struct {
 	y      uint16
 }
 
+func StdStreams() (stdOut io.Writer, stdErr io.Writer, stdIn io.ReadCloser) {
+	return os.Stdout, os.Stderr, os.Stdin
+}
+
+func GetHandleInfo(in interface{}) (uintptr, bool) {
```

Comment:
@sachin-jayant-joshi GetHandleInfo is a name that's too specific to windows. I suggest `Fd()`.